src/System.IO.FileSystem/tests/Directory/Exists.cs

```diff
@@ -381,6 +381,7 @@ public void SubdirectoryOnNotReadyDriveAsPath_ReturnsFalse()
         // We just care that we can access an accessible drive directly, we don't care which one.
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)] // drive labels
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotInAppContainer))] // Can't read root in appcontainer        
```

Comment:
For all of these where you added a ConditionalFact, you should also delete the Fact.